packaging_automation/tests/test_citus_package.py

```diff
@@ -24,8 +24,8 @@
     "debian/stretch": 4,
     "debian/buster": 4,
     "ubuntu/xenial": 2,
-    "ubuntu/bionic": 2,
-    "ubuntu/focal": 2
+    "ubuntu/bionic": 4,
+    "ubuntu/focal": 4
```

Comment:
Why did these lines change now? Did we suddenly start to create debug packages? Why is `debian/xenial` still 2?

Reply:
https://github.com/citusdata/packaging/issues/732

Reply:
Ubuntu 16.04 EOL'd 5 years after its release: April, 2021.

We can drop packaging support, and test coverage for them, I think.

Reply:
Opened an issue for this. We have download numbers. I think first we need to check download numbers and afterwars we need to declare it to community when we will drop support https://github.com/citusdata/tools/issues/170